tests/observability/metrics/utils.py

```diff
@@ -695,8 +695,8 @@ def create_windows11_wsl2_vm(
         name=vm_name,
         namespace=namespace,
         client=client,
-        vm_instance_type=VirtualMachineClusterInstancetype(name="u1.xlarge"),
-        vm_preference=VirtualMachineClusterPreference(name="windows.11"),
+        vm_instance_type=VirtualMachineClusterInstancetype(client=client, name="u1.xlarge"),
+        vm_preference=VirtualMachineClusterPreference(client=client, name="windows.11"),
```

Comment:
a user can create a vm with unprivileged_client; if unprivileged_client is passed as client, this will fail.